ci-operator/step-registry/openshift-qe/cluster-density-v2/openshift-qe-cluster-density-v2-commands.sh

```diff
@@ -31,10 +31,41 @@ iteration_multiplier=$(($ITERATION_MULTIPLIER_ENV))
 export ITERATIONS=$(($iteration_multiplier*$current_worker_count))
 
 export ES_SERVER="https://$ES_USERNAME:$ES_PASSWORD@search-ocp-qe-perf-scale-test-elk-hcm7wtsqpxy7xogbu72bor4uve.us-east-1.es.amazonaws.com"
-export EXTRA_FLAGS+=" --gc-metrics=true --profile-type=${PROFILE_TYPE}"
+export EXTRA_FLAGS+=" --gc-metrics=true --profile-type=${PROFILE_TYPE} --local-indexing"
```

Comment:
This will impact all the other jobs. Since this is specific to horreum poc, plese move this to poc specific if block.